CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 # Usage:
 # cmake [-D LLVM_DEV_DIR=$LLVM_DEV_DIR] $SAS_DIR
 
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
Ah, that was for the long strings in the test regexes; 3.0 supports line continuation with backslash. Actually though I decided to concatenate a list instead so this isn't necessary anymore.
________________________________
From: Joschka Lingemann [notifications@github.com]
Sent: 25 August 2016 11:28
To: jlingema/SAS
Cc: David Ho; Author
Subject: Re: [jlingema/SAS] Jlingema wip (#7)


In CMakeLists.txt<https://github.com/jlingema/SAS/pull/7#discussion_r76209762>:

> @@ -8,7 +8,7 @@
>  # Usage:
>  # cmake [-D LLVM_DEV_DIR=$LLVM_DEV_DIR] $SAS_DIR
>
> -cmake_minimum_required (VERSION 2.8)
> +cmake_minimum_required (VERSION 3.0)


Why do we require 3.0 now?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/jlingema/SAS/pull/7/files/7d30532a13617c228a6c513cc9a20498bcb6a12b#r76209762>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATV9_7Z9_-Nk9Z5GrDxTqiLE41gmhS3uks5qjWA3gaJpZM4Js3i6>.